examples/chip-tool/commands/clusters/WriteAttributeCommand.h

```diff
@@ -105,6 +110,38 @@ class WriteAttribute : public ModelCommand, public chip::app::WriteClient::Callb
         return mWriteClient->SendWriteRequest(device->GetSecureSession().Value());
     }
 
+    template <class T>
+    CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex, chip::NodeId senderNodeId,
```

Comment:
The function above then should be forcibly error'ing out if it encounters a group session, since this new function is now responsible for group comms.

Please add comments appropriately above.

Reply:
Pretty sure that if we pass a yaml test to chip-tool, we're still using the function above to send groups commands. @vivien-apple ?

Reply:
If a change was made so that it can handle groupSession it most probably is because a test somewhere was using it. As the goal of the PR was to enable group command for users that will input them manually I would defer this change in a future PR.